File: CPP/is_prime_number.cpp
#include<cmath>

bool isPrime(int num) {
  if (num == 0 || num == 1) 
    return false;

  int counter = 0;

  for (int i = 1; i <= sqrt(num); i++)
  {
    if (num % i == 0) 
      counter++;
    else
      continue;
  }

  if (counter == 1) 
    return true;

  return false;
}

File: Python/is_prime_number.py
import math


def is_prime(num):
    if (num <= 0 or num == 1): return False

    counter = 0

    for i in range(1, int(math.sqrt(num)) + 1):
        if (num % i == 0):
            counter += 1
        else:
            continue

    if (counter == 1): return True

    return False